src/main/java/ca/on/oicr/gsi/dimsum/controller/MvcExceptionHandler.java

```diff
@@ -14,32 +14,39 @@ public class MvcExceptionHandler {
 
     @Autowired
     private CommonModelAttributeProvider commonModelAttributeProvider;
+
     private static final Logger logger = LoggerFactory.getLogger(MvcExceptionHandler.class);
 
     @ExceptionHandler(ResponseStatusException.class)
     public ModelAndView handleResponseStatusException(ResponseStatusException ex) {
-        ModelAndView mav = new ModelAndView("error");
-        mav.addObject("errorStatus", ex.getStatus().toString());
-        mav.addObject("errorMessage", ex.getReason());
-
-        mav.addObject("dataAgeMinutes", commonModelAttributeProvider.getDataAgeMinutes());
-        mav.addObject("buildVersion", commonModelAttributeProvider.getBuildVersion());
+        ModelAndView mav = prepareErrorModelAndView(ex.getStatus().value(), ex.getReason(), ex);
 
         if (ex.getStatus().is5xxServerError()) {
-            logger.error(ex.getReason(), ex);
+            logger.error("Error Status: " + ex.getStatus() + ". Error Message: " + ex.getReason(),
+                    ex);
+        } else {
+            logger.info("Error Status: " + ex.getStatus() + ". Error Message: " + ex.getReason());
         }
```

Comment:
Move logging into `prepareErrorModelAndView` - it should be handled the same for both